test/Grpc.AspNetCore.Server.Tests/HttpContextStreamWriterTests.cs

```diff
@@ -107,9 +106,9 @@ public async Task WriteAsync_BufferHintWriteOptions_DoesNotFlush()
             var pipeReader = new StreamPipeReader(ms);
 
             var writtenMessage1 = await MessageHelpers.AssertReadStreamMessageAsync<HelloReply>(pipeReader);
-            Assert.AreEqual("Hello world 1", writtenMessage1.Message);
+            Assert.AreEqual("Hello world 1", writtenMessage1!.Message);
```

Comment:
Should this be ? instead of ! 

It seems to me the AssertReadStreamMessageAsync's API's intention is to return null if no message is read. In that case we should use ? right? 

This appears in a lot of places.

Reply:
I know in the test it is not null so this is to suppress the warning. It *could be* ? instead but meh. Either way it fails on null.